pkgs/development/python-modules/graspy/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytest
+, pytestcov
+, matplotlib
+, networkx
+, numpy
+, scikitlearn
+, scipy
+, seaborn
+}:
+
+buildPythonPackage rec {
+  pname = "graspy";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "neurodata";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1ss7d71lwblimg7ri88ir9w59j0ri13wl75091hjf7q0mchqr6yd";
+  };
+
+  checkInputs = [ pytest pytestcov ];
+
+  checkPhase = ''
+    runHook preCheck
+    # `test_autogmm` takes too long; fixed in next release (graspy/pull/328)
+    pytest tests -k 'not test_autogmm'
+    runHook postCheck
```

since the checkhooks are usually very specific to the application, you don't really need to call the hooks, but they are harmless to leave there.
```suggestion
    # `test_autogmm` takes too long; fixed in next release (graspy/pull/328)
    pytest tests -k 'not test_autogmm'
```